CONTRIBUTING.md

```diff
@@ -7,11 +7,16 @@ understand the choices made by the project.
 
 ## Run the APIv1 test (Python)
 
+:warning: Make sure to follow first the installation instructions in the README!
```

Comment:
What is missing if you don't follow the instructions? I wonder if we should bother with installing the service for running those tests.